src/envoy/http/backend_auth/filter.cc

```diff
@@ -107,8 +109,6 @@ void Filter::rejectRequest(Envoy::Http::Code code, absl::string_view error_msg,
   ENVOY_LOG(debug, "{}", error_msg);
   decoder_callbacks_->sendLocalReply(code, error_msg, nullptr, absl::nullopt,
                                      details);
-  decoder_callbacks_->streamInfo().setResponseFlag(
```

Comment:
In L85, for the case `token found`, the response flag is not set.

Can we always put the `setResponseFlag` in `rejectRequest` like filters ?

Reply:
But we don't know which ResponseFlag to set.

Reply:
Where do these response flags show up? Do our clients care about them? Seems like we dont need them

Reply:
They can be seen in the access logger.